python/pyspark/sql/tests/test_udf.py

```diff
@@ -616,6 +616,20 @@ def f():
 
         self.spark.range(1).select(f()).collect()
 
+    def test_worker_original_stdin_closed(self):
+        # Test the original stdin of worker inherit from daemon is closed
```

Comment:
Let's try to make it complete since this PR is rather a core fix.

```
Test if it closes the original standard input of worker inherited from the daemon,
and replaces it with '/dev/null'.  See SPARK-26175.
```